xbmc/interfaces/legacy/ListItem.cpp

```diff
@@ -857,7 +857,7 @@ namespace XBMCAddon
       for (const auto& el : alt.later())
       {
         if (el.which() == second)
-          throw WrongTypeException(StringUtils::Format("When using \"%s\" you need to supply a string or list of strings for the value in the dictionary", tag.c_str()).c_str());
+          throw WrongTypeException("%s", StringUtils::Format("When using \"%s\" you need to supply a string or list of strings for the value in the dictionary", tag.c_str()).c_str());
```

Comment:
Is there a benefit from formatting with `StringUtils::Format` and pass the result in a new format string instead of just calling `WrongTypeException` with the format string that is passed to `StringUtils::Format`?